python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -12,7 +12,7 @@
 """
 
 import base64
-from Cookie import SimpleCookie
+from tank_vendor.shotgun_api3.lib.six.moves.http_cookies import SimpleCookie
```

Comment:
The saml_core stuff is not part of Toolkit. You should contact @patrick-hubert-adsk and coordinate with him for those changes. Note that saml_sso doesn't have a dependency on Shotgun, so we'll have to probably port it to Python 3 with custom code. :(